doc/source/fparser2.rst

```diff
@@ -495,6 +495,16 @@ file was found but would fail if the include file was not found::
   program x
   include 'endprogram.inc'
 
+Compiler/OpenMP Directive support
+---------------------------------
+Most Fortran compilers support directives to enable compiler-specific
+functionality. Fparser will convert these into ``Directive`` nodes where
```

Comment:
I suggest we probably want this to say "optionally" and specify how the user turns this on/off. I'll start a discussion on the PR to get people's views. (At bare minimum I think it already requires the user to have requested that comments be retained by the reader?)